src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfSummaryRequestHandlerTests.cs

```diff
@@ -104,11 +105,14 @@ public async Task HandleAsync_GivenRequest_ShouldCheckOrganisationAccess()
         [Fact]
         public async Task HandleAsync_GivenRequest_SummaryTotalsShouldBeRetrieved()
         {
+            //arrange
+            short currentYear = (short)SystemTime.Now.Year;
```

Comment:
use fixture.Create<int>() trather than SystemTime.Now.Year. It allows us to check that the Request Compliance year is used in the tests rather than system time.